flow.js

```diff
@@ -172,6 +173,10 @@ class Flow {
     }
   }
   handleChange = (input, value, state) => {
+    let repairProblemChanged = false;
```

Comment:
Setting `repairProblemChanged` can be a single line, i.e.
```suggestion
    let repairProblemChanged = input == 'repairProblem' && state.data[input] != value;
```